dockers/docker-orchagent/tunnel_packet_handler.py

```diff
@@ -29,6 +30,9 @@
 
 STATE_DB = 'STATE_DB'
 APPL_DB = 'APPL_DB'
+COUNTERS_DB = 'COUNTERS_DB'
+TUNNEL_PKT_COUNTER_TEMPLATE = 'COUNTERS{}IPINIP_TUNNEL'
```

Comment:
These IPinIP packets are only kernel processed and dropped, should we add 'CPU' in this counter name? It might be confusing with the hardware based ipinip tunnel for which we don't have counters.